README.md

```diff
@@ -1,6 +1,6 @@
 # python3-unittest-runner
 
-This is a Python3 runner that executes tests based on the `unittest` module.
+This is a Python3 runner that executes tests based on the `unittest` module. If a `requirements.txt` file is detected at the root of the project, the dependencies will be automatically downloaded and installed.
```

Comment:
If `requirements.txt` doesn't exist, your build will fail :grin: 

You should add a check in the build script